packages/tools/remark-plugins/package.json

```diff
@@ -6,7 +6,8 @@
   "type": "module",
   "scripts": {
     "lint": "eslint src",
-    "build": "tsc"
+    "build": "tsc",
```

Comment:
not sure if it will work out but you could try to use `heft build` (see other packages in the lib)

Reply:
@webpro could you please check comment from Andy regarding the build as you worked on this plugin. I just added the `test` command to fix failing of the tests